### PR TITLE
Unlisted addons can be reviewed in the editor tools (bug 1144708)

### PR DESCRIPTION
**Pull request by magopian**
Fixes [bug 1144708](https://bugzilla.mozilla.org/show_bug.cgi?id=1144708)

This PR is based on PR #489, which is based on PR #487, which is based on PR #484, which is based on #479, only the last commit is relevant to this bug

### CODE REVIEW

File: apps/zadmin/tests/test_views.py [hunk 1]
```diff
@@ -1439,6 +1439,10 @@ def setUp(self):
         self.url = reverse('zadmin.addon_manage', args=[self.addon.slug])
         self.client.login(username='admin@mozilla.com', password='password')
 
+    def test_can_manage_unlisted_addons(self):
+        """Unlisted addons can be managed too."""
+        assert self.client.get(self.url).status_code == 200
```
Comment (diox) [inline apps/zadmin/tests/test_views.py]: Aren't you missing a `self.addon.update(listed=False)` ?

Comment (magopian) [inline apps/zadmin/tests/test_views.py]: I'm pretty sure I am indeed Oo

File: apps/editors/tests/test_views.py [hunk 2]
```diff
@@ -2046,7 +2046,14 @@ def test_action_links_as_admin(self):
         ]
         check_links(expected, pq(r.content)('#actions-addon a'), verify=False)
 
-    def test_action_links_as_admin_unlisted_addon(self):
+    def test_unlisted_addon_action_links(self):
+        """No "View Listing" link for the standard reviewers."""
```
Comment (diox) [inline apps/editors/tests/test_views.py]: Not a fan of the wording. Maybe something like `No "View Listing" link for unlisted addons, and not action links for standard reviewers`. And then below `No "View Listing" link for unlisted addons, and "Edit"/"Manage" links for admins` ? To make it clear that nobody is going to get `View Listing` link for unlisted Addons anyway, and only admins can see Edit/Manage.

**Comment by diox** [comment]
r+

**Pull request merged**